testsuite/model/src/test/java/org/keycloak/testsuite/model/session/OfflineSessionPersistenceTest.java

```diff
@@ -235,53 +237,63 @@ public void testPersistenceMultipleNodesClientSessionAtSameNode() throws Interru
     @RequireProvider(UserSessionPersisterProvider.class)
     @RequireProvider(value = UserSessionProvider.class, only = InfinispanUserSessionProviderFactory.PROVIDER_ID)
     public void testPersistenceMultipleNodesClientSessionsAtRandomNode() throws InterruptedException {
-        List<String> clientIds = withRealm(realmId, (session, realm) -> IntStream.range(0, 5)
-              .mapToObj(cid -> session.clients().addClient(realm, "client-" + cid))
-              .map(ClientModel::getId)
-              .collect(Collectors.toList()));
-        List<String> offlineSessionIds = createOfflineSessions(realmId, userIds);
+        try {
+            // increase session owners to 3 for the test
+            // test restarts nodes in cluster, and from time to time, it can happen that both owners for a session are offline/initializing
```

Comment:
Are you sure this is true? If this was true making the node restart should make this issue more reproducible. Hav you tried to make the restarting more often and run some stress test on GHA?

Reply:
It is not that simple. Having more restarts makes the cluster consistently broken. See  https://github.com/martin-kanis/keycloak/actions/runs/6470856387/